src/patternfly/components/ModalBox/examples/modal-box-no-header-example.hbs

```diff
@@ -0,0 +1,21 @@
+{{#> modal-box modal-box--attributes='aria-labelledby="modal-title"'}}
```

Comment:
Instead of `aria-labelledby`, this example should use `aria-label` instead. For example:

`{{#> modal-box modal-box--attributes='aria-label="Example of a modal without a header"'}}`